docs/skillmap/clicker/clicker1.md

```diff
@@ -73,8 +73,8 @@ scene.setBackgroundColor(11)
 ```
 
 ## Step 2
-Add a splash screen to let the user know to press the 
-Ⓐ button when they're ready to start playing.
+Add a [__*splash screen*__](#splasht "A full-screen message that shows while a program or level is loading") to let the user know to press the 
+**🅐** button when they're ready to start playing.
```

Comment:
not blocking for this update, but we should test these emojis with @livcheerful's screen reader changes--i'm not sure how the reader handles them

Reply:
I just tried it out -- It will display correctly, but won't be read out loud

Reply:
Ah yes, I remember this kind of issue with Code.org, but their solution won't work here.

Does the screen reader read the definition text or the alternate text on photos?

Reply:
And...if we had just a couple of these symbols that we used everywhere (A, B, Refresh) then could we code those in?

Reply:
I initially just removed all the image text because it seemed like neither text was meant to be read, but I can change it. And yeah we can do replacements for all symbols!